lib/fog/openstack/models/volume/snapshots.rb

```diff
@@ -0,0 +1,27 @@
+require 'fog/openstack/models/collection'
+
+module Fog
+  module Volume
+    class OpenStack
+      module Snapshots
+
+        def all(options = {})
+          load_response(service.list_snapshots_detailed(options), 'snapshots')
+        end
+
+        def summary(options = {})
+          load_response(service.list_snapshots(options), 'snapshots')
+        end
+
+        def get(snapshots_id)
+          if snapshot = service.get_snapshot_details(snapshots_id).body['snapshot']
+            new(snapshot)
+          end
+        rescue Fog::Volume::OpenStack::NotFound
+          nil
+        end
+
```

Comment:
Same empty line cosmetic comment. Just to be consistent, most of the time it's not there. I know most of the legacy stuff is not aligned too :(